packages/upload-api/src/access/confirm.js

```diff
@@ -73,7 +73,7 @@ export async function confirm({ capability, invocation }, ctx) {
   // Store the delegations so that they can be pulled with access/claim
   // The fact that we're storing proofs chains that we pulled from the
   // database is not great, but it's a tradeoff we're making for now.
-  await ctx.delegationsStorage.putMany(delegation, attestation)
+  await ctx.delegationsStorage.putMany(invocation.asCID, delegation, attestation)
```

Comment:
```suggestion
  await ctx.delegationsStorage.putMany(invocation.link(), delegation, attestation)
```

I would avoid `asCID`, which I should probably mark as deprecated in the ucanto.